app/views/base/layout.scala

```diff
@@ -198,6 +198,11 @@ object layout {
     """<svg width="0" height="0"><mask id="mask"><path fill="#fff" stroke="#fff" stroke-linejoin="round" d="M38.956.5c-3.53.418-6.452.902-9.286 2.984C5.534 1.786-.692 18.533.68 29.364 3.493 50.214 31.918 55.785 41.329 41.7c-7.444 7.696-19.276 8.752-28.323 3.084C3.959 39.116-.506 27.392 4.683 17.567 9.873 7.742 18.996 4.535 29.03 6.405c2.43-1.418 5.225-3.22 7.655-3.187l-1.694 4.86 12.752 21.37c-.439 5.654-5.459 6.112-5.459 6.112-.574-1.47-1.634-2.942-4.842-6.036-3.207-3.094-17.465-10.177-15.788-16.207-2.001 6.967 10.311 14.152 14.04 17.663 3.73 3.51 5.426 6.04 5.795 6.756 0 0 9.392-2.504 7.838-8.927L37.4 7.171z"/></mask></svg>"""
   )
 
+  private val prefersReducedMotionStyle = raw(
+    "<style type='text/css'> @media (prefers-reduced-motion: reduce) { *, *::before, *::after { "
+    + "animation: none !important; transition: none !important; } } </style>"
+  )
+
```

Comment:
why the new `<style>` tag in every page? why can't this be in the existing page stylesheet?

Reply:
I was having trouble overriding the topnav hamburger specificity to shut that animation down, and it degenerated to mysticism about rule definition order (turned out to just be a before/after thing).  At some point I moved style I was still tweaking to layout.scala to reduce turnaround on builds by a few mills and never moved it back to scss.